package.json

```diff
@@ -37,7 +37,7 @@
     "coverage": "open coverage/index.html",
     "lint": "npm run prettier:check && npm run tslint",
     "lint:fix": "npm run prettier:fix && npm run tslint:fix",
-    "tslint": "tslint -c tslint.full.json --project tsconfig.json --type-check",
+    "tslint": "tslint -c tslint.full.json --project tsconfig.json",
```

Comment:
Hey @raymondfeng, I am concerned about the effects of this change. Could you please explain why are you removing this flag?

In my understanding, `--type-check` is required to allow TSLint to apply advanced rules relying on type information, basically everything in [tslint.full.json](https://github.com/strongloop/loopback-next/blob/c6867e711cb2e04391165df50df7f57b276a90c0/tslint.full.json). I am worrying that by removing `--type-check`, we are effectively disabling this rules.

PTAL.